hermes.podspec

```diff
@@ -11,8 +11,12 @@ module HermesHelper
     "command -v #{bin} > /dev/null 2>&1"
   end
 
+  def self.cmake_configuration
+    "-DHERMES_ENABLE_DEBUGGER:BOOLEAN=true -DHERMES_ENABLE_FUZZING:BOOLEAN=false -DHERMES_ENABLE_TEST_SUITE:BOOLEAN=false"
+  end
+
   def self.configure_command
-    "./utils/build/configure.py #{BUILD_TYPE == :release ? "--distribute" : "--build-type=Debug"} --cmake-flags='-DHERMES_ENABLE_DEBUGGER:BOOLEAN=true -DHERMES_ENABLE_FUZZING:BOOLEAN=false -DHERMES_ENABLE_TEST_SUITE:BOOLEAN=false -DCMAKE_INSTALL_PREFIX:PATH=../destroot' build"
+    "./utils/build/configure.py #{BUILD_TYPE == :release ? "--distribute" : "--build-type=Debug"} --cmake-flags='#{HermesHelper.cmake_configuration} -DCMAKE_INSTALL_PREFIX:PATH=../destroot' build"
```

Comment:
Since I barely remember what little I ever knew about cocoapods, how do all the pieces here come together?

If I understand correctly, the plan is that we will provide an npm which includes the prebuilt hermes pod, and this will be consumed by mac apps which use hermes.  Is this right?

I see where circleci builds the pod above, but I also see code below (lines 44,46) where the same build commands appear in the podspec.  When are these commands used?

For an application which consumes hermes, how does cocoapods know where to find the prebuilt hermes framework?

Reply:
As mentioned above, this is here so users can install from source, if need be. The releases we cut just come with the artefacts in the same place as where this podspec expects them to be (e.g. `vendored_framework`)

Reply:
And correct, the plan is to ship them in npm bundles, as most other RN deps are distributed. (I am personally not convinced we should keep doing it in such a roundabout way, but for now it works.)

Reply:
Oh, I think I see.  The app Podspec would point to `node_modules/hermes-engine-darwin` (maybe with some `..`s), then? I think that was the bit I hadn't figured out.

Reply:
Yep, that’s it 👍 See the integration test for an example of what that syntax would look like (although in that test the binaries don’t exist yet and so it will build them through cmake) https://github.com/facebook/hermes/blob/f128811121ce711cd0b99c2abe7dfd649168cca3/test/ApplePlatformsIntegrationTestApp/Podfile#L6

Reply:
For posterity, there’s no real need to distribute this as a npm package, but we’re doing this simply for consistency with the other RN deps (as discussed with @wilholen).